server/src/main/java/com/distkv/server/storeserver/StoreServer.java

```diff
@@ -85,7 +107,9 @@ public static void main(String[] args) {
     if (listeningPort > 0) {
       config.setPort(listeningPort);
     }
+
     StoreServer storeServer = new StoreServer(config);
+    storeServer.config();
```

Comment:
What do you think of putting this line in the constructor?
I don't think that the life cycle needs to be called by the user himself. When the user uses it, he can check it by himself.

Reply:
these interface will not exposed to user.

Reply:
I know, I just discuss this from the api level, but this is not a big problem.

Reply:
I know, I just discuss this from the api level, but this is not a big problem.

Reply:
I know, I just discuss this from the api level, but this is not a big problem.

Reply:
I know, I just discuss this from the api level, but this is not a big problem.